HelpSource/scdoc.css

```diff
@@ -1,11 +1,13 @@
+
 body {
-    font-family: Arial, Helvetica, sans-serif;
+    font-family: Helvetica, Arial, sans-serif;
     font-size: 10pt;
     color: black;
     background: white;
 /*    max-width: 60em;*/
 /*    margin-left: 20em;*/
-    padding: 0;
+    width: auto;
+    padding-left: 1.5em;
```

Comment:
`width: auto` adds a horizontal scrollbar to some pages (e.g. the Array helpfile for me). `padding-left: 1.5em` adds whitespace to the left of the top bar so it looks "broken off."